.env.example

```diff
@@ -37,6 +37,5 @@ RELAYER_STORAGE_PATH=storage/leveldb
 RELAYER_QUERIES_TASK_QUEUE_CAPACITY=10000
 RELAYER_CHECK_SUBMITTED_TX_STATUS_DELAY=10
 
-#LOGGER_LEVEL=info
-#LOGGER_OUTPUTPATHS=stdout, /tmp/logs
```

Comment:
wait, we can't specify logger outputs now?

Reply:
i'm not sure that i like the idea of forcing the user to use both env vars and config files if they want to set logger output. also, you'll need to mount additional volumes if you want to do that in docker, which is sort of dirty

Reply:
we could, but the realisation was so-so. improved it so the logger is configurable via envs again